src/main/java/org/symphonyoss/symphony/messageml/markdown/nodes/form/FormElementNode.java

```diff
@@ -40,7 +40,4 @@ public String getText() {
     return text;
   }
 
-  public void setText(String text) {
-    this.text = text;
-  }
```

Comment:
is never used in the code so I removed it

Reply:
It is not possible to retrieve a *Node ? Keep in mind that this is an open source library and we don't know how it is used

Reply:
Do we need a method to set the text of a Node? the text should be coming directly from the corresponding element. This can create a difference between the text inside the messageML/presentationML and the markdown